packages/venia-ui/lib/components/MegaMenu/megaMenuItem.js

```diff
@@ -63,32 +62,34 @@ const MegaMenuItem = props => {
         ) : null;
     }, [category, isFocused, mainNavWidth, subMenuState, handleCloseSubMenu]);
 
+    const maybeDownArrowIcon = category.children.length ? (
+        <Icon
+            className={classes.arrowDown}
+            src={ArrowDown}
+            size={16}
+            aria-label={
+                'Category: ' +
+                category.name +
+                '. ' +
+                category.children.length +
+                ' sub-categories'
+            }
+        />
+    ) : null;
+
     return (
         <div className={megaMenuItemClassname}>
             <Link
                 onKeyDown={e => {
-                    a11yClick(e) && toggleSubMenu(e);
+                    handleKeyDown(e);
```

Comment:
You shouldn't need to do your own binding here unless you're passing additional parameters. This can be:
`onKeyDown={handleKeyDown}`

Reply:
Fixed in 08332ac